assets/queries/k8s/non_kube_system_pod_with_host_mount/query.rego

```diff
@@ -0,0 +1,74 @@
+package Cx
+
+CxPolicy [result] {
+  resource := input.document[i]
+  metadata := resource.metadata
+  not metadata.namespace
+  kind := resource.kind
```

Comment:
FYI, if you just use it once, you don't need to define a variable. Just a heads up, don't change if you don't want to.

Reply:
ok!